doc/source/rllib-training.rst

```diff
@@ -520,6 +520,142 @@ Custom metrics can be accessed and visualized like any other training result:
 
 .. image:: custom_metric.png
 
+Customized Exploration Behavior (Training and Evaluation)
```

Comment:
Should add a link in `rllib-toc.rst` to this too (the TOC has to be manually updated).

Reply:
+1